# Translation and Protein Biosynthesis
## Summary

In gene expression, genetic information flows from DNA to RNA to protein. This is also referred to as the central dogma of molecular biology. The process of translating DNA into RNA is called transcription, while protein synthesis based on the RNA template is called translation. Details on gene expression and transcription can be found in the chapter "Gene Expression and Transcription."

The translation takes place at the ribosomes - large molecular complexes made of ribosomal RNA (rRNA) and proteins. The ribosomes bind the RNA template, which is also referred to as messenger RNA (mRNA), and catalyze the formation of a polypeptide based on this template. They progressively link one amino acid to the next. In addition to mRNA and rRNA, transfer RNAs (tRNAs) are important RNAs of translation. They form the adapter molecules that carry out the actual translation of the mRNA sequence into the amino acid sequence of the synthesized proteins: They recognize the base triplets on the mRNA and supply the corresponding amino acids for incorporation into the peptide chain. The translation ends when a specific base sequence on the mRNA is reached. The ribosome then dissociates; the mRNA and the newly synthesized protein are released.

Already during translation, proteins begin to fold into their correct three-dimensional structure. Only when they are properly folded can they perform their function. Various specialized proteins assist the emerging proteins in developing their structure or ensure that a protein is functional after translation by modifying certain amino acid side chains. This also includes ensuring that the protein reaches the right location within the cell, such as the cell membrane or the mitochondria.

The translation rate is adjusted to the prevailing cellular conditions; for example, the presence or absence of certain nutrients can influence the translation rate.
## Principle of Protein Biosynthesis

- Translation: Translation of the base sequence of the mRNA into the amino acid sequence of a polypeptide using tRNA
    - Location: Ribosomes in the cytosol and on the rough ER
    - Phases
        - Initiation (Translation)
            - Start of translation by assembling the ribosomal subunits
            - Recognition of the start codon by the start tRNA with the help of initiation factors
        - Elongation (Translation): Synthesis of proteins (this is where the formation of a peptide bond occurs, catalyzed by the ribosome (Ribozym!))
        - Termination (Translation)
            - Termination of protein synthesis at the stop codon
            - Dissociation of the ribosome
            - Release of mRNA and synthesized protein

## Basics: Genetic Code, RNAs, and Ribosomes

### The Genetic Code

- Definition: Relationship between the base sequence of DNA (or mRNA) and its corresponding amino acid sequence in a protein
- Properties
    - Codons as units: Three bases (triplet) form a codon and encode for an amino acid.
        - Codon: Sequence on the mRNA that consists of three bases and codes for a specific amino acid.
        - Anticodon: Sequence on the tRNA that serves as a recognition site for the mRNA template and consists of three bases.
            - The anticodon is complementary to the codon on the mRNA
            - Codons and anticodons always pair antiparallel: The base at the 5' end of a codon pairs with the base at the 3' end of the anticodon and vice versa.
    - The genetic code is "degenerate": This means that most amino acids are coded by multiple codons, not just one.
        - Examples
            - The amino acid isoleucine can be encoded by the triplets AUA, AUC, and AUU.
            - The amino acids arginine, serine, and leucine are each encoded by six triplets!
            - Only methionine and tryptophan each have their own specific triplet.
    - Universality
        - The genetic code is almost universal among all organisms.
        - However, there are deviations; for example, some triplets are used differently in mitochondria than in the cell nucleus.
- Start codon: AUG
    - The codon AUG serves as a signal that the mRNA sequence is starting to be translated.
    - Encodes simultaneously for the amino acid methionine, so that each polypeptide starts with methionine.
- Stop codons: UGA, UAG, and UAA terminate translation.
    - Special feature of the incorporation of selenocysteine
        - The so-called 21st amino acid selenocysteine is incorporated at the codon UGA when the mRNA forms a specific secondary structure at that position.
        - Most of the time, UGA is read as a stop codon.
- Wobble phenomenon (engl. wobble for "to sway, to wobble")
    - Definition: Some tRNA molecules recognize several different codons through a certain inaccuracy ("wobble") in the pairing of the first base of the anticodon with the third base of the codon. This is also referred to as the wobble position at the 5' position of the anticodon.
    - The base at the 5' end of the anticodon is often inosine instead of one of the usual bases.
        - Inosin pairs relatively unspecifically with uridine, adenosine, and cytidine at the 3' end of the codon.
        - They form so-called wobble base pairs.

> [!note]
> The stop codon UGA also codes for the amino acid selenocysteine under certain conditions!

> [!note]
> Selenocysteine is incorporated into the growing polypeptide chain at the ribosome!

> [!note]
> To remember the stop codons UGA, UAG, and UAA, it helps to use: U Go Away, U Are Gone, and U Are Away!

### Involved Ribonucleic Acids (RNAs)

- RNAs form the essential components of translation.
    - mRNA: Template for protein synthesis (contains the codons)
    - tRNA: Adapter between mRNA and amino acids (contains the anticodons)
    - rRNA: Structural and catalytic components of ribosomes

For an overview of the structure and function of RNAs, see also: Classification of RNA

#### Charging of tRNAs

In order to perform their function as adapter molecules, tRNAs must be loaded with the amino acid specific to them.

- Definition: Linking of a tRNA with its specific amino acid
- Enzyme: Various aminoacyl-tRNA synthetases specific to each amino acid (For each proteinogenic amino acid, there exists at least one specific synthetase that recognizes the "correct" amino acid for its anticodon)
- Reaction mechanism: The charging of tRNAs in the cytosol occurs in two steps, which are catalyzed by the same enzyme.
    - Activation of the amino acid with ATP consumption: Aminoacyl-AMP is formed.
        - Amino acid + ATP → Aminoacyl-AMP + PPi
    - Transfer of the activated amino acid to the tRNA and formation of an ester bond
        - tRNA + Aminoacyl-AMP → Aminoacyl-tRNA + AMP

### Ribosome Structure

Ribosomes are the organelles of protein biosynthesis. They are primarily composed of ribonucleic acids, but also of many different proteins. For an overview of ribosomes as organelles, see also: The Cell.

- The structure of ribosomes is very complex; for a rough understanding of their function, the following positions are important:
    - Binding site for the mRNA on the small ribosomal subunit
    - Binding sites for the tRNAs are formed jointly by both ribosomal subunits.
        - A(Aminoacyl)-site
        - P(Peptidyl)-site
        - E(Exit)-point
    - Peptidyl transferase center
        - Catalytic center (on the large ribosomal subunit)
        - Forms the peptide bond between the amino acids that are bound to the A and P sites (each via tRNA)
    - Exit tunnel: Protein exit channel (within the large ribosomal subunit)
    - Binding sites for regulatory factors

## Process of Translation in Eukaryotes

The translation is divided into three phases: Initiation, Elongation, and Termination.

### Initiation of the Translation

- Definition: Assembly of the functional ribosome and recognition of the start codon by the first tRNA
- Involved molecules
    - 40S and 60S subunit of the ribosome
    - Mature mRNA
    - Charged Start-tRNA (Initiator Methionyl-tRNA)
    - Energy supplier: GTP
    - Eukaryotic initiation factors (eIF), among others:
        - eIF-2
            - Belongs to the small G-proteins
            - Binds the initiator tRNA in its active, GTP-bound form (so-called ternary complex)
            - Contributes to the formation of the completed initiation complex by hydrolyzing GTP to GDP
            - Is converted back into the GTP-bound form by the guanine nucleotide exchange factor eIF-2B.
        - eIF-4: Recognizes mRNA
- Process of Initiation
    1. The small ribosomal subunit associates with specific initiation factors.
    2. The ternary complex of methionine-loaded start tRNA, initiation factor eIF-2, and GTP binds to the small ribosomal subunit, thus forming the 43S pre-initiation complex.
    3. The mRNA is recognized by the initiation factor eIF-4 and binds to the 43S pre-initiation complex, resulting in the formation of the 48S pre-initiation complex.
        1. The initiator tRNA recognizes the start codon (usually the first AUG triplet after the 5' cap of the mRNA); GTP hydrolysis provides the energy for the release of eIF-2.
    4. The large and small ribosomal subunits assemble into the 80S ribosome, the completed initiation complex.

### Elongation of Translation: Protein Biosynthesis

- Principle
    - The mRNA is read from the 5' to the 3' direction (triplet by triplet)
    - The protein is built up amino acid by amino acid: Through the attack of the free NH2 group of one amino acid on the esterified COOH group of another amino acid, a peptide bond is formed between the COOH group of the growing peptide chain and the NH2 group of the incoming amino acid.
    - Elongation begins at the N-terminus and proceeds towards the C-terminus: The N-terminus of the growing protein thus leaves the ribosome first.
- Process of Elongation
    - Initiator methionyl-tRNA is located at the peptidyl site (P site) or another previously suitable aminoacyl-tRNA is bound there.
    - Binding of the next aminoacyl-tRNA to the aminoacyl site (A site) of the ribosome
        - The anticodon of the aminoacyl-tRNA must match the codon on the mRNA that will be read next.
        - The eukaryotic elongation factor eEF-1 is involved in the binding, which is activated by GTP.
    - Linking of amino acids at the P-site and A-site of the ribosome via a peptide bond
        - Catalyzed by the peptidyl transferase activity of the ribosome's large subunit
    - Translocation
        - The ribosome moves along the mRNA by the length of a triplet
        - The energy comes from the hydrolysis of GTP with the involvement of eEF-2.
        - After translocation, the tRNA that was at the A site is now at the P site, and the tRNA that was previously at the P site is at the exit site (E site).
            - The A site is freed up again and can bind a new aminoacyl-tRNA (in complex with eEF-1/GTP)
            - At the P site, there is a tRNA with the growing peptide chain.
            - Throughout the entire translation, the growing polypeptide chain is covalently attached to a tRNA molecule at its C-terminus (as a peptidyl-tRNA molecule)
            - The empty tRNA is located at the E site.
    - Release of the uncharged tRNA from the E-site


> [!note] Diphtheria
> Diphtheria is a potentially life-threatening infectious disease that primarily affects the upper respiratory tract and is caused by the toxin-producing Corynebacterium diphtheriae. The diphtheria toxin leads to the inactivation of the eukaryotic elongation factor 2 (eEF-2), which inhibits protein biosynthesis and causes the affected cell to die. Due to very high diphtheria vaccination rates among children in Germany, the incidence is extremely low in this country.

### Termination of the Translation

- Termination of protein synthesis at a stop codon
- Release of the protein
- A termination factor helps with this by recognizing stop codons and hydrolytically cleaving the peptidyl-tRNA bond.

## Protein Folding and Misfolding of Proteins

In order for a protein to perform its function within the cell or organism, it must be correctly folded. This means that the protein takes on a specific three-dimensional structure. The folding of a protein begins during translation. The spatial structure is determined by the amino acid sequence of the protein (for details on spatial structure and denaturation of proteins, see also: amino acids and proteins).

### Protein Folding

- Definition: Training of the three-dimensional structure of a protein from the unfolded to the native state.
- Principle: Progressive stabilization of intermediate stages until the energetically most favorable stage is reached.
    - Hydrophobic collapse
        - Accumulation of hydrophobic side chains of amino acids, such as valine, leucine, or isoleucine, in the interior of a protein during folding
        - This effect influences the folding process and can be regarded as its driving force.
- Influence of folding by:
    - Formation of disulfide bridges: During folding, disulfide bridges form between two cysteine residues in some (mostly extracellular) proteins.
    - Covalent modifications: E.g. phosphorylation of tyrosine side chains or glycosylation of asparagine side chains
- Chaperone: Proteins that assist other proteins in forming the correct structure
    - Protein-Disulfide-Isomerase: Helps to form the thermodynamically most favorable disulfide bridges within a protein (in proteins that have more than two cysteine residues), in case energetically less favorable disulfide bridges have already formed.
    - Prolyl-cis-trans-Isomerase
        - Helps to find the energetically most favorable conformation of a peptide bond involving a proline residue.
        - Usually, the trans conformation is preferred, but sometimes the cis conformation is more favorable.
    - Chaperone (from French chaperon = "matron")
        - Definition
            - Protein complexes that prevent the aggregation of proteins during synthesis and can refold misfolded proteins in a protected environment.
            - A variety of chaperones consume ATP in the process.
        - Families e.g.
            - Hsp70 (Heat shock protein 70)
            - Hsp60 (Chaperonine)
            - Hsp90
        - Function
            - In particular, prevention of protein aggregation by the binding of chaperones to exposed hydrophobic regions of a protein that is currently being synthesized at the ribosome.
            - Additionally, support in the transport of proteins across intracellular membranes as well as in the degradation of proteins
        - Mode of action using BiP as an example
            - BiP is a chaperone in the ER
            - His KDEL sorting signal determines the retention in the ER and is recognized by the so-called COP coat of transport vesicles.
            - COPs (COPI and COPII) are among those responsible for transport processes between the ER, Golgi apparatus, and cell membrane.
                - If a defect disrupts the return of BiP to the ER, extracellular aggregation occurs.
        - Mechanism of action using the example of the heat shock protein Hsp60
            - Hsp60 protein complexes form a barrel-like structure into which fully synthesized, but misfolded or only partially folded proteins can be incorporated. Hsp60 recognizes misfolded proteins by normally non-exposed hydrophobic surfaces in the protein.
            - The co-chaperone Hsp10 closes the chamber under ATP binding like a lid.
            - The enclosed protein can fold into its energetically favorable conformation under the protected conditions.
            - The protein is released after hydrolysis of ATP, as Hsp10 dissociates from Hsp60 and the chamber opens.

> [!note]
> The three-dimensional structure of a protein is encoded in its amino acid sequence! Nevertheless, some proteins require assistance in achieving the correct folding. This task is performed by chaperone proteins!

### Misfolding of Proteins

Misfolded proteins can cause damage both intracellularly and extracellularly. Therefore, they are typically either assisted by chaperone proteins in adopting the correct folding or recognized, tagged, and proteolytically degraded by specialized proteins (for protein degradation see also: metabolism of amino acids and proteins).

- Definition: Missing folding or non-native folding of a protein caused by denaturing influences.
- Triggering factors
    - Stress, e.g. oxidative conditions
    - Elevated body temperature (fever)
    - Mutations, e.g. misfolding of the chloride channel protein in cystic fibrosis, misfolding of hemoglobin in sickle cell anemia
- Mechanism of misfolding
    - In particular, hydrophobic amino acid residues on the surface of a misfolded protein tend to aggregate with other hydrophobic surfaces/proteins.
    - Aggregations of β-sheet-rich proteins can also lead to fibril formation.
- Intracellular reaction
    - Misfolded proteins are recognized and either rescued by chaperones or ubiquitinated and thus marked for degradation in the proteasome.
- Consequences of Misfolding
    - In some diseases, the protective mechanisms fail.
    - Misfolded proteins then form insoluble aggregates and fibrils that lead to cell death (e.g., in the neurodegenerative diseases Parkinson's disease and Alzheimer's disease)


> [!note] Cystic fibrosis
> In cystic fibrosis, a metabolic disorder also known as Mukoviszidose, a specific mutation in the gene coding for the CFTR (cystic fibrosis transmembrane regulator) chloride channel is very often present. Due to the deletion of a base triplet that encodes phenylalanine at position 508, the channel protein is not correctly folded and is directed to degradation in the proteasome. Therefore, the channel protein, which would actually be functional despite the deletion, does not reach the cell membrane. As a result, only hyperviscous secretions are secreted by the exocrine glands, leading to chronic inflammatory reactions and organ damage.

## Protein modification

The functionality of many proteins is not only dependent on them being correctly folded. They often undergo specific chemical changes during or after translation, known as co- or post-translational modifications. For example, mature proteins can be formed by cleavage from a precursor protein or by the oxidation of cysteine residues to disulfide bridges. Specific side chains can also be modified, for instance, by the addition of sugar residues, hydrophobic groups, or (reversibly) various functional groups.

### Protein Glycosylation

Some proteins contain covalently bound carbohydrate groups (oligosaccharides). They are then referred to as glycoproteins. Often, these are proteins of the cell membrane or secretory proteins, e.g., serum proteins such as erythropoietin.

- Definition: Enzymatic attachment of carbohydrate residues to specific amino acid side chains of proteins, resulting in the formation of N-glycosidic (N-glycosylation) or O-glycosidic bonds (O-glycosylation)
- Location: Depending on the type of glycosylation in the rough endoplasmic reticulum and/or Golgi apparatus
- Function: Glycosylation affects the
    - Spatial structure and localization of a protein and thereby its function
    - Stability and lifespan of a protein
        - Example: Serum glycoproteins
            - Terminal N-acetylneuraminic acid (sialic acid) present: No degradation in the hepatocytes
            - Loss of N-acetylneuraminic acid: The oligosaccharides of glycoproteins end with galactose or N-acetylgalactosamine, are recognized by asialoglycoprotein receptors on hepatocytes, taken up via receptor-mediated endocytosis, and degraded lysosomally.
- Mechanism of N-glycosylation
    - Transfer of a large oligosaccharide synthesized at the ER membrane to an asparagine residue in the protein to be glycosylated.
    - Transport into the Golgi apparatus and modification of the oligosaccharide (so-called trimming)

|                             | N-Glycosylation                                                             | O-Glycosylation                    |
| --------------------------- | --------------------------------------------------------------------------- | ---------------------------------- |
| Location                    | - Start in the rough ER: Binding of oligosaccharides to asparagine residues | - Completed in the Golgi apparatus |
|                             | - Continuation in the Golgi apparatus: Trimming                             |                                    |
| Share of all glycosylations | approx. 90%                                                                 | approx. 10%                        |
| Linkage via                 | N-glycosidic bond                                                           | O-glycosidic bond                  |
| Involved Amino Acid         | Asparagine                                                                  | Serine or Threonine                |
| Carbohydrate side chains    | Complex structured oligosaccharides                                         | Less complex oligosaccharides      |
| Involved Sugar Residues     | - Glucose                                                                   | - Galactose                        |
|                             | - Mannose                                                                   | - N-Acetylgalactosamine            |
|                             | - Fucose                                                                    | - N-Acetylglucosamine              |
|                             | - Galactose                                                                 | - N-Acetylneuraminic Acid          |
|                             | - N-Acetylglucosamine                                                       |                                    |
|                             | - N-Acetylneuraminic Acid (Sialic Acid)                                     |                                    |
| Enzyme/involved molecules   | - Specific glycosyltransferases                                             | - Specific glycosyltransferases    |
|                             | - Dolicholphosphate (in the membrane)                                       |                                    |
|                             | - Other enzymes                                                             |                                    |
> [!note]
> The N-glycosylation, that is the attachment of sugars to asparagine residues of proteins, begins in the rough ER!

> [!note]
> The sugar residue N-acetylneuraminic acid (sialic acid) protects glycoproteins from degradation by the liver!

> [!note]
> The enzymatic glycosylation should not be confused with non-enzymatic glycation: In glycation, aldoses like glucose spontaneously bind with their carbonyl group in a Schiff base reaction to the amino groups of proteins, potentially impairing their function! Glycated proteins undergo further chemical rearrangement processes and are then referred to as AGEs (Advanced Glycation End Products).

> [!note]
> Pay attention to which compartment the modification takes place in. N-glycosylation primarily occurs in the endoplasmic reticulum, while O-glycosylation is localized in the Golgi apparatus.

> [!note] HbA1c
> Hemoglobin (Hb) is responsible for binding the transported oxygen in erythrocytes. In particular, the N-terminus of the β-chains of Hb is non-enzymatically glycosylated at elevated blood sugar levels. The resulting HbA1c (glycosylated hemoglobin type A1) is functionally unaffected and persists for the entire lifespan of the erythrocyte (8-12 weeks). The so-called HbA1c value indicates the average blood sugar level over this period and can therefore be used for monitoring the progression of diabetes mellitus.

### Lipid Anchor

Most membrane proteins interact with lipid membranes through hydrophobic side chains (e.g., valine or leucine residues). However, there are also other ways for proteins to bind to membranes, namely through so-called lipid anchors that are covalently attached to the protein.

- Definition: Covalent modification of proteins with hydrophobic groups, through which proteins are anchored in membranes.
- Types
    - Acylation: Linking with long-chain fatty acids, e.g., palmitic acid
    - Isoprenylation: Linking a cysteine side chain of the protein with a polyisoprene via a thioether bond, as seen in the
        - Farnesylation: Linkage with a farnesyl group (three isoprene units, a total of 15 carbon atoms)
            - Example: RAS protein
        - Geranylgeranylation: Linking with a geranylgeranyl group (four isoprene units, a total of 20 carbon atoms)
            - Example: γ-subunit of heterotrimeric G-proteins
    - GPI anchor: Linkage to glycosylphosphatidylinositol (GPI), a glycolipid

### Sulfation

Some proteins, including certain cell surface proteins, blood coagulation factors, and chemokine receptors, are sulfated.

- Definition: Binding of a sulfo group to the OH group of a tyrosine by sulfotransferases in the Golgi apparatus.
- Goal: Increase water solubility, involvement in the functional effects of the proteins

### Reversible Protein Modification

Reversible modification of proteins by specific enzymes is a way to regulate the activity or stability of proteins or to transmit signals. Most of the time, the spatial structure (more precisely, the conformation) of the proteins changes due to the modification. As a result, they can, for example, recognize substrates and/or interact with other proteins. Due to the reversibility of the modification, proteins can be "turned on or off" - not only immediately after translation but at any time and depending on the respective cellular conditions.

- Phosphorylation: The attachment of phosphate groups to the hydroxyl group (OH group) of serine or threonine residues, or of tyrosine residues by kinases.
    - Example: Regulation of glycogen phosphorylase in glycogen metabolism through phosphorylation
    - Antonym: Dephosphorylation
- Acetylation: Linking with a CO-CH3 group through protein acetyltransferases
    - Example: Regulation of DNA condensation and thus of transcription through acetylation of histone proteins
    - Opposite: Deacetylation
- Succinylation: Linking a lysine side chain with a succinyl molecule
    - Example: Modification of histone proteins
- Ubiquitination: Linking the C-terminus of ubiquitin (a small protein) with the ε-amino group of lysine residues in proteins, especially in proteins that are to be degraded.
    - Usually, whole chains of ubiquitin are attached.
    - Example: Regulation of the cell cycle through ubiquitination and degradation of cyclins
- ADP-ribosylation: Transfer of an ADP-ribose moiety from NAD+ by an ADP-ribosyltransferase
    - Example: Histone modification through poly-ADP-ribosylation
- Sumoylation: Attachment of a SUMO (Small Ubiquitin-like Modifier) protein to lysine residues (similar to ubiquitination)
    - Example: Many regulators of the cell cycle are themselves regulated by SUMOylation.
    - Opposite: DeSUMOylation

## Protein Sorting

Not all proteins fulfill their function in the cytosol: Many proteins need to be transported to other cellular compartments, such as the mitochondria, the nucleus, or the cell membrane, or they are secreted.

- Definition: Distribution of proteins to their destinations during or after translation based on signal sequences.
- Mechanisms
    - There are various mechanisms by which proteins reach their target location.
    - A signal sequence at the N-terminus of a protein is usually responsible for initiating the correct cellular processes.
        - Basically, one distinguishes:
            - Protein synthesis on free ribosomes: Cytosolic proteins and proteins destined for transport into the mitochondria, peroxisomes, or the cell nucleus.
            - Protein synthesis on ribosomes of the rough ER: Membrane proteins, lysosomal, and secretory proteins are synthesized at the rough ER (described in more detail below).

### Translocation of Ribosomes to the Rough ER

Proteins that are to leave the cell (secretory proteins, e.g., fibronectin), as well as membrane proteins and lysosomal proteins, are first synthesized on free ribosomes in the cytosol. However, their synthesis is halted shortly after it begins, and the ribosome is brought to the cytosolic side of the rough ER. Here, protein synthesis is continued, and the protein is synthesized directly into the lumen of the ER.

#### Mechanism

1. Start of translation on free ribosomes in the cytosol
2. When a signal sequence (a specific amino acid sequence of 9 to 12 amino acids) is synthesized, it is bound by the so-called signal recognition particle (SRP, a ribonucleoprotein).
3. SRP halts translation and brings the ribosome with the peptide chain to the ER membrane.
4. The ER membrane contains the SRP receptor, to which the SRP binds along with the ribosome and the nascent peptide chain.
5. The SRP and the SRP receptor both have GTP bound, which they then hydrolyze to GDP → SRP is released and can bind a new signal sequence.
6. The ribosome is transferred to the so-called translocon - a protein complex that spans the ER membrane, which then opens like a channel.
7. The translation continues and the protein is synthesized directly into the ER lumen.
8. Even during translation, the signal sequence of the growing protein is cleaved off by a signal peptidase.
9. After the termination of translation, the ribosome is released back into the cytosol.
10. The channel of the translocon closes and the fully synthesized protein is in the ER.
    - Already during translation, the protein has folded into its native structure, among other things with the help of chaperones (see above).

#### Further Process

- Protein modification in the ER
    - N-glycosylation
    - Formation of disulfide bridges
- Distribution of the proteins
    - Proteins for the cell membrane are anchored directly in the ER membrane.
        - Process: They usually contain hydrophobic sections in their amino acid sequence that are integrated into the ER membrane.
    - Soluble proteins for the lysosomes or those that are to leave the cell via exocytosis remain in the ER lumen until they are transported after translation is complete.
    - Through transport vesicles, all proteins newly synthesized in the ER are delivered to their destination via the Golgi network.
    - The lysosomal proteins receive a mannose-6-phosphate residue, which is recognized and bound by the membrane-bound mannose-6-phosphate receptor in the trans-Golgi network, ensuring the transport of these proteins in vesicles to the lysosomes.
    - Soluble proteins of the mitochondrial matrix or inner mitochondrial membrane are characterized by an N-terminal signal sequence that forms a positively charged, amphipathic α-helix.
    - Soluble proteins that are to remain in the ER are marked by the signal sequence KDEL (according to the one-letter code, the sequence stands for the amino acids Lysine-Aspartate-Glutamate-Leucine) at their C-terminus.
    - Only properly folded proteins can leave the ER towards their destination.
        - Otherwise, they are bound by chaperones and transported back into the cytosol, where they are broken down.

## Regulation of Translation

The regulation of gene expression at the level of translation allows for a faster cellular response to external factors than transcriptional regulatory mechanisms. The corresponding amounts of proteins in a cell can be regulated through an adjustment of the translation rate, for example, in response to altered nutrient supply, cellular stress, or during differentiation and development.

- Main focus: Initiation phase of translation (Regulation of the initiation factors)
- Regulatory mechanisms
    - Reduced formation of the ternary complex of eIF-2, GTP, and initiator tRNA due to phosphorylation of the initiation factor eIF-2
        - Phosphorylated eIF-2 in GDP-bound form can no longer be converted by eIF-2B into the active GTP-bound form, preventing the formation of the ternary complex necessary for the initiation of translation, resulting in a reduced translation rate.
            - Example: Synthesis of globin in erythrocytes
                - If there is not enough heme available to incorporate it into a globin protein, the synthesis of globin is inhibited by the phosphorylation of eIF-2.
    - Regulation on the Cap Recognition Process by eIF-4
        - Example: The protein kinase mTOR (mammalian Target of Rapamycin) generally increases protein biosynthesis in response to increased nutrient availability (especially amino acids) and plays a key role in the regulation of cell growth.
            - mTOR phosphorylates, among other things, eIF-4E-binding proteins (4E-BPs), for example in response to growth factors and cytokines, thereby promoting the initiation of translation.

> [!note] Rapamycin
> Rapamycin (also known as Sirolimus) is an immunosuppressant that inhibits the protein kinase mTOR. It thereby inhibits the growth and differentiation of lymphocytes. Due to its immunosuppressive effect, rapamycin is used in organ transplants (especially in kidney transplants). Since mTOR is involved in many signaling pathways associated with cell growth, the growth-inhibiting effect of rapamycin is also utilized elsewhere: In coronary artery disease, drug-eluting stents coated with rapamycin are often used in the narrowed coronary arteries to reduce the risk of a rapid re-narrowing within the stent.

## Review Questions on the Chapter Translation and Protein Biosynthesis

### Basics: Genetic Code, RNAs, and Ribosomes

What is referred to as the genetic code and what does it mean that it is "degenerate"? Provide an example!
- Under the term "genetic code," it is understood that the base sequences of DNA or mRNA encode specific amino acid sequences. Its unit is the codon (a base triplet). Since the genetic code is "degenerate," most amino acids can be encoded by several different codons. For example, the amino acids arginine, serine, and leucine are each encoded by six different triplets.

What do the terms codon and anticodon stand for, and how do these nucleotide sequences interact with each other?
- The codon is a sequence of three bases (triplet) of the mRNA, which encodes for an amino acid. In contrast, the anticodon refers to a base triplet on the tRNA, which is exactly complementary to the codon on the mRNA! The base at the 5'-end of a codon always pairs with the base at the 3'-end of the anticodon and vice versa (antiparallel pairing).

What special feature is there in the encoding of selenocysteine?
- The 21st amino acid selenocysteine is incorporated into the polypeptide chain at the ribosome during translation, like all other amino acids. However, there is a peculiarity that selenocysteine is encoded by the codon UGA, which usually functions as a stop codon. When the mRNA forms a specific secondary structure (so-called hairpin structure) at the 3'-end of the mRNA, selenocysteine is incorporated. Otherwise, UGA is read as a stop codon, and translation is terminated at that point.

Why does every polypeptide begin with methionine and what usually happens co-translationally with it?
- The start codon with the base sequence AUG signals that the mRNA sequence is now beginning to be translated. At the same time, it also encodes for the amino acid methionine, so that each polypeptide starts with methionine. Co-translationally, this N-terminal methionine is usually cleaved off by methionine aminopeptidase.

What is the so-called wobble phenomenon and which base do you associate with this term?
- Some tRNA molecules recognize several different codons due to a certain imprecision ("wobble") in the pairing of the first base of the anticodon with the third base of the codon. This is also referred to as the wobble position at the 5'-position of the anticodon. The base at the 5'-end of the anticodon of the tRNA is often inosine.

Describe the reaction mechanism during the loading of tRNA!
- The loading of the tRNA with its specific amino acid occurs in two steps in the cytosol. First, the amino acid is activated in an ATP-consuming process, resulting in the formation of aminoacyl-AMP. Subsequently, the activated amino acid is transferred to the tRNA, forming an ester bond. Both steps are catalyzed by an aminoacyl-tRNA synthetase specific for the respective amino acid. Eukaryotic cells thus have at least 20 (21 with selenocysteine) different synthetases that attach "their" specific amino acid to the respective tRNAs coding for it. For example, the arginyl-tRNA synthetase attaches arginine to all its tRNAs.

How are ribosomes generally structured and which important positions do you know?
- Ribosomes are primarily composed of ribonucleic acids and are very complex in structure. The binding site for the mRNA is located on the small ribosomal subunit. The peptidyl transferase center is located on the large ribosomal subunit and is the catalytic center. The binding sites for the tRNAs are formed by both ribosomal subunits and are referred to as the A, P, and E sites.
### Process of Translation in Eukaryotes

The translation process is well-known to occur in three phases: initiation, elongation, and termination. How does the eukaryotic initiation factor eIF-2 work?
- The eukaryotic initiation factor eIF-2 belongs to the small G-proteins. In its active, GTP-bound form, it binds the start tRNA (the so-called ternary complex). By the ternary complex attaching to the small ribosomal subunit, it helps prevent premature binding to the large subunit of the ribosome. The complete initiation complex is only formed when the start tRNA recognizes the start codon (usually the first AUG triplet after the 5'-cap of the mRNA) and eIF-2 subsequently hydrolyzes its bound GTP to GDP.

Describe the principle of elongation! How are the individual amino acids linked together?
- For protein synthesis, the base sequences of the mRNA are read from 5' to 3' direction and translated into amino acids, which are linked together. A peptide bond is formed between the growing peptide chain and the next amino acid, as the free NH2 group of the incoming amino acid attacks the esterified COOH group of the previous amino acid. Elongation begins at the N-terminus and proceeds toward the C-terminus, meaning that the N-terminus of the growing protein exits the ribosome first.

Describe the process of translocation during protein synthesis!
- During translocation, the ribosome moves along the mRNA by the length of one base triplet. The energy required for this comes from the hydrolysis of GTP with the involvement of eEF-2. After translocation, the tRNA that was previously located at the A site is now at the P site. The tRNA that was previously at the P site is now at the exit site (E site) and is released from there.

How is the translation completed?
- Termination of translation corresponds to the end of protein synthesis at the stop codon. It occurs with the help of a termination factor that recognizes stop codons and hydrolytically cleaves the peptidyl-tRNA bond. This results in the release of the synthesized protein.

### Protein Folding and Misfolding of Proteins

What is the purpose of the auxiliary protein protein disulfide isomerase in the context of protein folding?
- The protein disulfide isomerase aids during protein folding in forming the thermodynamically favorable disulfide bridges within a protein.

What is the purpose of chaperones in the context of protein folding?
- Chaperones are helper proteins that prevent the aggregation of proteins during their synthesis. Additionally, they can allow misfolded proteins to refold in a protected environment.

### Protein modification

Some proteins are modified by covalently attaching carbohydrate groups (oligosaccharides) to them. What are these proteins called?
- Proteins that contain covalently bound carbohydrate groups are referred to as glycoproteins. They are often proteins of the cell membrane or secretory proteins, such as erythropoietin.

What types of protein glycosylation do you know? Where do they occur?
- In N-glycosylation, carbohydrate residues are enzymatically attached to the asparagine side chains of proteins through the formation of an N-glycosidic bond. N-glycosylation begins with the attachment of large oligosaccharides in the rough ER and is continued with the so-called trimming in the Golgi apparatus. In contrast, O-glycosylation takes place completely in the Golgi apparatus.

Explain the significance of protein glycosylation using the example of N-acetylneuraminic acid from serum glycoproteins!
- The terminal N-acetylneuraminic acid (sialic acid) prevents the degradation of serum glycoproteins in the liver. Upon the loss of N-acetylneuraminic acid, the glycoproteins are recognized by asialoglycoprotein receptors on hepatocytes and are taken up and degraded via endocytosis.

How do glycosylation and glycation differ from each other? Name an example of the latter!
- Glycosylation occurs enzymatically, while glycation occurs non-enzymatically. In glycation, aldoses like glucose spontaneously bind to amino groups of proteins in a Schiff base reaction, which can impair their function. Glycated proteins, such as HbA1c, undergo further chemical rearrangement processes and are then also referred to as AGEs (Advanced Glycation End Products).

What is the purpose of lipid anchors? Name an exemplary example of the attachment of a lipid anchor!
- Through so-called lipid anchors, proteins can be anchored in the cell membrane. Various types are distinguished, including acylation, isoprenylation, and GPI anchors. For example, in farnesylation, the protein is linked to a farnesyl group through specific cysteine side chains.

What is meant by reversible protein modification and what is its purpose? Give an example!
- Reversible protein modification refers to the covalent attachment of various chemical groups to proteins, with the forward and reverse reactions being catalyzed by specific enzymes. This regulates the activity or stability of proteins. For instance, in phosphorylation, phosphate residues are attached to the hydroxyl group of serine, threonine, or tyrosine residues by kinases. In ADP-ribosylation, an ADP-ribose residue from NAD+ is transferred by an ADP-ribosyltransferase, for example, in histone modification through poly-ADP-ribosylation.

### Protein Sorting

Why does protein synthesis occur on free ribosomes on one hand, but also on ribosomes of the rough ER on the other hand?
- Cytosolic proteins as well as proteins for the mitochondria, peroxisomes, and nucleus are synthesized on free ribosomes. Likewise, the synthesis of secretory proteins, membrane proteins, and lysosomal proteins begins on the free ribosomes in the cytosol. Shortly after the synthesis begins, however, it is paused and the ribosome is brought to the rough ER. There, protein synthesis is continued, and the protein is synthesized directly into the lumen of the ER. In the rER, among other processes, protein modifications occur (N-glycosylation and formation of disulfide bridges), and the proteins are then forwarded to their destinations via vesicular transport.

Describe the mechanism of the translocation of ribosomes to the rough ER until the moment the ribosome binds to the ER!
- Translation begins at the free ribosomes in the cytosol. As soon as a signal sequence is synthesized, the so-called signal recognition particle (SRP) binds to it, halts translation, and brings the ribosome along with the peptide chain to the ER membrane. The ER membrane contains the SRP receptor, to which the SRP along with the ribosome and the nascent peptide chain binds. Subsequently, SRP is released through GTP hydrolysis and can bind a new signal sequence.

What allows lysosomal proteins to be recognized and transported to the lysosome?
- Lysosomal proteins receive a mannose-6-phosphate residue. This residue is recognized and bound by the membrane-bound mannose-6-phosphate receptor in the trans-Golgi network. It facilitates the transport of these proteins in vesicles to the lysosomes.

How can soluble proteins that have their destination in the endoplasmic reticulum (and are also supposed to remain there) be recognized?
- Soluble proteins that are to remain in the ER are marked by the signal sequence KDEL.

### Regulation of Translation

Through which regulatory mechanism can the translation rate be reduced by phosphorylation? Name an example!
- Through phosphorylation of the initiation factor eIF-2, the formation of the ternary complex composed of eIF-2, GTP, and start tRNA can be decreased, which reduces the translation rate. For example, if a erythrocyte does not have enough heme available to incorporate it into a globin protein, the synthesis of globin is inhibited by the phosphorylation of eIF-2.
