# Purines and Pyrimidines
## Summary

Purines and pyrimidines are chemical structures that form a fundamental component of nucleotides in DNA and RNA. Therefore, they are essential for information storage in the cell. They also serve as a framework for coenzymes and are thus involved in many enzymatic processes. A change in purine or pyrimidine metabolism can have very different consequences. For example, disturbances in purine metabolism lead to an increased amount of uric acid in the blood and can trigger gout. Inhibitors of nucleotide synthesis are used in tumor therapy, e.g., ribonucleotide reductase inhibitors, which hinder DNA replication in often dividing tumor cells due to a lack of DNA building blocks.

## Conversion of Ribonucleotides into Deoxyribonucleotides

Regardless of whether purine or pyrimidine nucleotides: Deoxyribonucleotides are required for DNA synthesis, specifically those containing the bases adenine, guanine, cytosine, and thymine. Apart from thymine-containing deoxyribonucleotides, the dNTPs (deoxyribonucleoside triphosphates) necessary for DNA can be formed by the reduction of their ribonucleotides. Ribonucleotide reductase catalyzes this reaction.

- Definition: Reduction of ribonucleotides to deoxyribonucleotides
- Goal: Provision of the DNA building blocks dATP, dGTP, dCTP (and indirectly also dTTP) in the S-phase of the cell cycle
- Summation equation: NDP (ribonucleoside diphosphate) + NADPH+H+ → dNDP + NADP+ + H2O
- Enzyme: Ribonucleotide reductase
- Other enzymes and cofactors: Thioredoxin, Thioredoxin reductase with the cofactor FAD, NADPH+H+
- Regulation: Complex regulation of activity and substrate specificity of ribonucleotide reductase, including the inhibition of the enzyme by dATP.

The reaction mechanism of the ribonucleotide reductase reaction involves the formation of free radicals. In the end, two cysteine residues of the ribonucleotide reductase are connected by a disulfide bridge (oxidized), allowing the ribose to be reduced. The O-atom at the C2' atom of the ribose residue leaves the nucleotide as H2O, taking with it two protons and two electrons from the enzyme. The deoxyribonucleotide is released.

- Additional reactions for restoring the original state of ribonucleotide reductase:
    1. Regeneration of the SH groups (thiol groups) of ribonucleotide reductase by thioredoxin
2. Thioredoxin contains its own SH groups, which are oxidized (thus forming a disulfide bridge).
    3. Regeneration of the SH groups of thioredoxin by the enzyme thioredoxin reductase using FADH2, which is thereby converted to FAD.
    4. FAD is reduced to FADH2 by NADPH+H+, which simultaneously produces NADP+.
    5. NADPH+H+ comes from the pentose phosphate pathway.

> [!note]
> In the end, the reducing equivalents for the reduction of ribonucleotides to deoxyribonucleotides come from NADPH+H+.

> [!note]
> Thioredoxin reductase contains selenium in the form of selenocysteine!

## Purine Metabolism

Purine nucleotides can be newly synthesized in the body (de novo synthesis) or recycled from degradation products (salvage pathway). The new synthesis of purine scaffolds is very energy-intensive. Therefore, it is highly regulated. The recycling of purine bases, which are released during the degradation of nucleotides or absorbed through intestinal resorption, saves energy. The degradation of purine nucleotides leads through several intermediate steps to purine bases and ultimately to uric acid, which is primarily excreted via urine.

### De novo Synthesis of Purine Nucleotides

- Definition: Neosynthesis of purine nucleotides from 5-phosphoribosyl-1α-pyrophosphate (PRPP) in 10 reaction steps to inosine-5'-monophosphate (IMP), then in two additional steps to adenosine-5'-monophosphate (AMP) and guanosine-5'-monophosphate (GMP) respectively.
- Starting material: PRPP
- Nitrogen or carbon donors: 2 glutamines, glycine, aspartate, bicarbonate (HCO3‑), 2 tetrahydrofolates (N10-formyl-tetrahydrofolate)
- Energy consumption: For PRPP → AMP, 5 energy-rich compounds are split, for PRPP → GMP, 6 energy-rich compounds.
- Key enzyme: Glutamine-PRPP-Amidotransferase
- Regulation: Allosteric activation of the key enzyme by PRPP, allosteric inhibition by adenine and guanine nucleotides.

> [!note]
> Principle of Synthesis:  
> 5-Phosphoribosyl-1α-pyrophosphate (PRPP) → → → (in 10 steps to) Inosine-5'-monophosphate (IMP)
> IMP → → AMP or GMP
> AMP → ATP or GMP → GTP

#### Provision of PRPP

- Definition: Pyrophosphorylation of ribose-5-phosphate to 5-phosphoribosyl-1α-pyrophosphate (PRPP) with the consumption of one molecule of ATP
- Substrate: Ribose-5-phosphate (from the pentose phosphate pathway)
- Product: 5-Phosphoribosyl-1α-pyrophosphate (PRPP)
- Energy consumption: ATP → AMP
- Enzyme: Ribosephosphate pyrophosphokinase (also: 5-phosphoribosyl-1-pyrophosphate synthetase or PRPP synthetase)
- Result: An activated ribose derivative for the incorporation of nucleotide bases
#### Phase 1: Synthesis pathway from PRPP to IMP

Starting from PRPP, the atoms of the purine scaffold are gradually added in ten reaction steps by various nitrogen or carbon donors until the first common intermediate, the purine nucleotide IMP, is formed.

- Step rate enzyme: Glutamine-PRPP-amidotransferase
    - Substrate: PRPP, Glutamine
    - Product: 5-Phosphoribosylamine (PRA), Glutamate, PPi
    - Regulation:
        - Activation by PRPP
        - Inhibition by all adenine and guanine nucleotides, e.g. ATP

> [!note]
> The N9 atom, which forms the N-glycosidic bond between the base and sugar in the newly synthesized purine nucleotide, comes from glutamine!

| Reactions                                                    | Substrate                                                       | Enzyme                          | Product                                                                                                                         | Special Features/Reagents                              |
| ------------------------------------------------------------ | --------------------------------------------------------------- | ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
| 1. Incorporation of the N9 atom                              | 5-Phosphoribosyl-1α-pyrophosphate (PRPP)                        | Glutamine-PRPP Amidotransferase | 5-Phosphoribosylamine (PRA)                                                                                                     | - Glutamine → Glutamate + PPi                          |
| 2. Incorporation of atoms C4, C5, and N7                     | 5-Phosphoribosylamine (PRA)                                     | GAR Synthetase                  | Glycinamide Ribonuclotide (GAR) also: 5-Phosphoribosylglycinamide                                                               | - + Glycine<br>- ATP → ADP + Pi                        |
| 3. Incorporation of the C8 atom                              | Glycinamide ribonucleotide (GAR)                                | GAR transformylase              | Formylglycinamide ribonucleotide (FGAR) also: 5'-phosphoribosylformylglycinamide                                                | - N10-formyl-tetrahydrofolate → tetrahydrofolate (THF) |
| 4. Incorporation of the N3 atom and amidation of the C4 atom | Formylglycinamide ribonucleotide (FGAR)                         | FGAM synthetase                 | Formylglycinamidin ribonucleotide (FGAM)                                                                                        | - Glutamine → Glutamate<br>- ATP → ADP + Pi            |
| 5. Closure for the formation of the imidazole ring           | Formylglycinamidine-Ribonucleotide (FGAM)                       | AIR synthetase                  | 5-Aminoimidazole-Ribonucleotide (AIR) also: 5'-Phosphoribosyl-5-Aminoimidazole                                                  | - ATP → ADP + Pi                                       |
| 6. Incorporation of the C6 atom                              | 5-Aminoimidazole-Ribonucleotide (AIR)                           | AIR-Carboxylase                 | 4-Carboxy-5-Aminoimidazole-Ribonucleotide (CAIR) also: 5'-Phosphoribosyl-5-Aminoimidazole-4-Carboxylic acid                     | - + HCO3-                                              |
| 7. Incorporation of the N1 atom (1st step)                   | 4-Carboxy-5-Aminoimidazole-Ribonucleotide (CAIR)                | SAICAR Synthase                 | 5-Aminoimidazole-4-N-Succinocarboxamide-Ribonucleotide (SAICAR) also: 5'-Phosphoribosyl-5-Aminoimidazole-4-N-Succinocarboxamide | - + Aspartate<br>- ATP → ADP + Pi                      |
| 8. Incorporation of the N1 atom (Step 2)                     | 5-Aminoimidazole-4-N-Succinocarboxamide-Ribonucleotide (SAICAR) | Adenylosuccinate Lyase          | 5-Aminoimidazole-4-Carboxamide-Ribonucleotide (AICAR) also: 5'-Phosphoribosyl-5-Aminoimidazole-4-Carboxamide                    | - Fumarate is released                                 |
| 9. Addition of the C2 atom                                   | 5-Aminoimidazole-4-Carboxamide-Ribonucleotide (AICAR)           | AICAR-Transformylase            | 5-Formamidoimidazole-4-Carboxamide-Ribonucleotide (FAICAR) also: 5'-Phosphoribosyl-5-Formamidoimidazole-4-Carboxamide           | - N10-Formyl-Tetrahydrofolate → Tetrahydrofolate (THF) |
| 10. Closure through the elimination of H2O                   | 5-Formamidoimidazole-4-Carboxamide-Ribonucleotide (FAICAR)      | IMP-Cyclohydrolase              | Inosine-5'-monophosphate (IMP) also: Inosinate                                                                                  | - H2O is eliminated                                    |

#### Phase 2: Synthesis pathway from IMP to AMP or from IMP to GMP

- AMP biosynthesis: Exchange of the oxygen atom at the C6 atom of IMP for an amino group (NH2 group)
    - 1. Reaction step: Inosin-5'-monophosphate (IMP) + Aspartate + GTP → Adenylosuccinate + GDP + Pi
        - Enzyme: Adenylosuccinate Synthetase
    - 2. Reaction step: Adenylosuccinate → Adenosine-5'-monophosphate (AMP) + Fumarate
        - Enzyme: Adenylosuccinate Lyase
- GMP biosynthesis: Attaching an amino group to the C2 atom
    - 1. Reaction step: Inosine-5'-monophosphate (IMP) + H2O + NAD+ → Xanthosine monophosphate (XMP) + NADH + H+
        - Enzyme: IMP-Dehydrogenase
    - 2. Reaction step: XMP + ATP + Glutamine → Guanosine-5'-monophosphate (GMP) + AMP + PPi + Glutamate
        - Enzyme: Xanthylate Aminase

> [!note]
> GTP is involved in the synthesis of AMP from IMP, while ATP is involved in the synthesis of GMP from IMP!

> [!note]
> Aspartate serves as an amino group donor in the synthesis of AMP from IMP!

#### Phase 3: Phosphorylation of AMP and GMP

The monophosphates AMP and GMP are phosphorylated to the corresponding diphosphates (ADP, GDP) and triphosphates (ATP, GTP). Various kinases can catalyze this reaction.

### Salvage of Purine Bases (Salvage Pathway)

Free purine bases can be directly converted into purine nucleotides in the body with PRPP. This pathway of purine nucleotide synthesis is associated with significantly lower energy consumption than de novo synthesis.

- Definition: In the salvage pathway, the purine bases adenine, guanine, and hypoxanthine are recycled.
- Substrate: PRPP with adenine or PRPP with guanine and hypoxanthine
- Product: AMP or GMP and IMP (Inosine Monophosphate)
- Enzyme: Adenine Phosphoribosyltransferase (APRT) and Hypoxanthine-Guanine Phosphoribosyltransferase (HGPRT)
- Regulation: Inhibition of APRT by adenine nucleotides, inhibition of HGPRT by IMP and GMP

> [!note] Lesch-Nyhan syndrome
> Due to a genetic defect, the enzyme HGPRT is absent. Therefore, guanine and hypoxanthine cannot be converted into GMP and IMP with PRPP. The PRPP concentration in the cell increases, which activates the key enzyme of purine de novo synthesis, glutamine-PRPP amidotransferase. The increased purine synthesis also leads to more purines being converted into uric acid. This results in hyperuricemia, possibly accompanied by gout symptoms and neurological disorders, including delayed mental development and auto-aggressive behavior. The treatment is symptomatic, inhibiting the conversion of purines into uric acid with medication.

### Degradation of Purine Nucleotides

Purines are broken down through different reaction steps than those used for their synthesis. Since the purine ring structure in the human organism cannot be enzymatically cleaved, purines are converted into uric acid and excreted in urine.

- Definition: Breakdown of purine nucleotides to their corresponding bases followed by oxidation of the formed xanthine to uric acid.
- Reaction steps in the degradation of AMP:
    - AMP → Adenosine → Inosine → Hypoxanthine → Xanthine → Uric acid
- Reaction steps in the degradation of GMP:
    - GMP → Guanosine → Guanine → Xanthine → Uric acid
- Reaction steps in the degradation of XMP (Xanthosine monophosphate):
    - XMP → Xanthosine → Xanthine → Uric acid
- Important enzymes
    - Adenosine deaminase: Catalyzes the deamination of adenosine to inosine
    - AMP-Deaminase: Catalyzes the deamination of AMP to IMP
    - Xanthine oxidase (also xanthine dehydrogenase): Catalyzes the conversion of hypoxanthine to xanthine and from xanthine to uric acid.
    - Guanindesaminase: Catalyzes the deamination of guanine to xanthine

> [!note] Principle of dismantling: 
> Purinmononucleotide → Purine nucleoside + Phosphate
> Purine nucleoside → Purine base + Ribose-1-phosphate
> Both adenosine and guanosine are ultimately converted into uric acid via xanthine, which is predominantly excreted in the urine.

> [!note]
> Uric acid is present in the blood as urate (i.e., in deprotonated form) and is excreted in this form in the urine!

> [!note] Hyperuricemia and Gout
> Uric acid as the end product of the breakdown of purine nucleotides is relatively poorly soluble in blood. The serum uric acid level is normally around 6.4 mg/dL (approximately 380 μmol/L). An increase in the serum uric acid level is referred to as hyperuricemia. Due to its poor solubility, excess uric acid can deposit in the form of sodium urate crystals in joints, connective tissue, ear cartilage, and in the renal medulla. This condition is known as gout, which is associated with acute inflammatory episodes and severe pain. Therapeutically, allopurinol is used, among other treatments. The structural analog of hypoxanthine inhibits xanthine oxidase. As a result, less uric acid is produced, and hypoxanthine and xanthine are excreted as end products of purine metabolism. The formation of uric acid stones is favored by a low pH, therefore stone formation in the urine can also be counteracted by alkalinization.

## Pyrimidine Metabolism

Pyrimidine nucleotides are synthesized anew or recycled just like purine nucleotides. Unlike purine nucleotides, in pyrimidine nucleotides, the basic structure of the base is built first during de novo synthesis and then connected with activated ribose phosphate.

### De novo synthesis of pyrimidine nucleotides

The pyrimidine nucleotides contain the bases uracil, cytosine, and thymine. In de novo synthesis, UMP (uridine-5'-monophosphate) is first formed. From this, CTP (cytidine-5'-triphosphate) can be synthesized via UTP (uridine-5'-triphosphate). For the synthesis of thymine-containing deoxyribonucleotides, a "detour" is necessary. Uracil-containing deoxyribonucleotides must first be formed as starting material. This reaction is catalyzed by ribonucleotide reductase. After several intermediate steps, dUMP can finally be methylated to dTMP (deoxythymidine-5'-monophosphate), also known as thymidylate. This reaction, catalyzed by thymidylate synthase, requires N5-N10-methylenetetrahydrofolate as a cofactor.

#### Phase 1: Synthesis pathway from aspartate and carbamoyl phosphate to UMP

- Definition: Neosynthesis of the pyrimidine backbone from aspartate and carbamoyl phosphate and transfer to PRPP (5-phosphoribosyl-1α-pyrophosphate) with subsequent release of CO2 through three enzyme complexes in the cytosol and at the inner mitochondrial membrane up to uridine-5'-monophosphate (UMP)
- Nitrogen, carbon, and oxygen donors: Glutamine (N-donor: ammonia), bicarbonate (C- and O-donor), aspartate (C- and N-donor: complete molecule)
- Involved enzyme complexes
    - CAD enzyme (localized in the cytosol)
    - Dihydroorotate dehydrogenase (located in the inner mitochondrial membrane)
    - UMP-Synthase (localized in the cytosol)
- Key reaction: Aspartate + Carbamoyl phosphate → N-Carbamoylaspartate + Pi
- Regulation: Activation of CAD by PRPP, inhibition of CAD by UTP, inhibition of the dihydroorotase activity of CAD and dihydroorotate dehydrogenase by orotate.

> [!note] Principle of Synthesis:  
> Glutamine + HCO3- + 2 ATP + H2O → Carbamoyl phosphate + Glutamate + 2 ADP + Pi
> Carbamoyl phosphate + Aspartate → N-Carbamoylaspartate + Pi
> N-Carbamoylaspartate → Dihydroorotate → Orotate
> Orotic acid + 5-Phosphoribosyl-1α-pyrophosphate (PRPP) → → (in 2 steps to) Uridine-5'-monophosphate (UMP)

> [!note]
> The cytosolic carbamoyl phosphate synthetase 2 of pyrimidine synthesis must not be confused with the mitochondrial carbamoyl phosphate synthetase 1 of the urea cycle!

| Synthesis of UMP                                                   |                                    |                                     |                                    |                                                                         |
| ------------------------------------------------------------------ | ---------------------------------- | ----------------------------------- | ---------------------------------- | ----------------------------------------------------------------------- |
| Reactions                                                          | Substrate                          | Enzyme                              | Product                            | Special Features/Reagents                                               |
| CAD-Enzyme (in the cytosol)                                        |                                    |                                     |                                    |                                                                         |
| 1. Reaction step                                                   | - Glutamine                        | - Carbamoyl phosphate synthetase 2  | - Carbamoyl phosphate              | - + HCO3-<br>- + H2O<br>- Glutamate is released<br>- 2 ATP → 2 ADP + Pi |
| 2. Reaction step                                                   | - Carbamoyl phosphate              | - Aspartate transcarbamylase        | - N-Carbamoylaspartate             | - + Aspartate<br>- Pi is released                                       |
| 3. Reaction step                                                   | - N-Carbamoylaspartate             | - Dihydroorotase                    | - Dihydroorotate                   | - H2O is released                                                       |
| Dihydroorotate dehydrogenase (in the inner mitochondrial membrane) |                                    |                                     |                                    |                                                                         |
| 4. Reaction step                                                   | - Dihydroorotate                   | - Dihydroorotate dehydrogenase      | - Orotate                          | - The enzyme is a flavoprotein (with FMN = flavin mononucleotide).      |
| UMP-Synthase (in the cytosol)                                      |                                    |                                     |                                    |                                                                         |
| 5. Reaction step                                                   | - Orotate                          | - Orotate phosphoribosyltransferase | - Orotidine-5'-monophosphate (OMP) | - PRPP → PPi                                                            |
| 6. Reaction step                                                   | - Orotidine-5'-monophosphate (OMP) | - OMP decarboxylase                 | - Uridine-5'-monophosphate (UMP)   | - CO2 is released                                                       |

#### Phase 2: From UMP to UTP to CTP

Kinasen phosphorylate UMP (uridine 5'-monophosphate) to UDP (uridine 5'-diphosphate) and finally to UTP (uridine 5'-triphosphate) - each time consuming ATP. From UTP, cytidine 5'-triphosphate (CTP) can be formed through amination. This step requires glutamine, H2O, and ATP. Glutamine serves as the donor of the necessary amino group. The responsible enzyme, cytidine triphosphate synthase (also: CTP synthase or CTP synthetase), is inhibited by CTP and activated by GTP.

> [!note]
> UMP → UDP → UTP → CTP

#### Phase 3: Synthesis of Thymine-containing Deoxyribonucleotides

- Definition: Formation of dTMP (thymidylate) from the precursor dUMP (deoxyuridine-5'-monophosphate) through methylation.
- Goal: Provision of thymidine-containing deoxyribonucleotides as DNA building blocks
- Summation equation: dUMP + N5-N10-Methylenetetrahydrofolate → dTMP + Dihydrofolate
- Enzyme: Thymidylate synthase
- Cofactor: N5-N10-Methylenetetrahydrofolate (a derivative of folic acid) as a methyl group carrier
- Additional reactions occurring for the regeneration of N5-N10-methylenetetrahydrofolate:
    - Dihydrofolate is converted into tetrahydrofolate by dihydrofolate reductase. NADPH+H+ is required as a cofactor.
    - Tetrahydrofolate is regenerated to N5-N10-methylenetetrahydrofolate.
- Subsequently: Phosphorylation of dTMP to dTDP and dTTP by specific kinases using ATP.

> [!note]
> The synthesis speed of thymine-containing deoxyribonucleotides is related to the availability of folic acid!

> [!note] Methotrexate and Fluorouracil
> Cancer cells proliferate very strongly, meaning they divide faster than other body cells. Their cellular metabolism is increased and the DNA synthesis rate is high. Chemotherapeutic agents curb the proliferation of cancer cells by, for example, slowing down DNA synthesis. Methotrexate (also known as Amethopterin) is a folic acid antagonist. It competitively inhibits dihydrofolate reductase and thus prevents the regeneration of tetrahydrofolate. As a result, the amount of thymidylate needed for DNA synthesis decreases in the cell. A lack of thymine leads to cell death through apoptosis. Fluorouracil is also a chemotherapeutic agent, more specifically a pyrimidine antagonist. It is converted intracellularly to fluorodeoxyuridine monophosphate (F-dUMP). F-dUMP inhibits thymidylate synthase and thus the formation of thymidylate. Additionally, phosphorylated fluorouracil is incorporated into RNA as fluoro-UTP instead of UTP.

### Reuse of Pyrimidine Nucleosides

Pyrimidine nucleosides can be recycled into pyrimidine nucleotides by kinases with the consumption of ATP. Free pyrimidine bases without a sugar moiety cannot be reused.

### Degradation of Pyrimidine Nucleotides

Pyrimidine nucleotides can, in contrast to purine nucleotides, be completely degraded and made available for metabolism for energy production. In the first step, the pyrimidine nucleotides are converted into their nucleosides, just like the purine nucleotides. After the cleavage of the sugar moiety, the free bases are reduced, and the pyrimidine ring can be cleaved in the next step. CO2 and NH3 are released, resulting in β-alanine and β-aminoisobutyrate. These compounds can be metabolized after conversion to malonyl-CoA and methylmalonyl-CoA, respectively.

- Definition: Degradation of pyrimidine nucleotides to β-alanine and β-aminoisobutyrate with intermediate steps involving the corresponding bases and hydrolytic cleavage of the pyrimidine ring.
- Location: Cytoplasm (especially of liver and kidney cells)
- Starting materials: CMP, UMP, and dTMP
- End products: β-Alanine and β-Aminoisobutyrate
- Reaction steps in the degradation of CMP
    - CMP → Cytidine → Uridine → Uracil → Dihydrouracil → β-Ureidopropionate → β-Alanine
    - CMP can also be converted into UMP and further broken down from there.
- Reaction steps in the degradation of UMP
    - UMP → Uridine → Uracil → Dihydrouracil → β-Ureidopropionate → β-Alanine
- Reaction steps in the degradation of dTMP
    - dTMP → Deoxythymidine → Thymine → Dihydrothymine → β-Ureidoisobutyrate → β-Aminoisobutyrate

## Review Questions on the Chapter Purines and Pyrimidines

### Conversion of Ribonucleotides into Deoxyribonucleotides

Describe the reduction of ribonucleotides to deoxyribonucleotides, including the enzymes involved! What is the purpose of this reaction?
- By the reduction of ribonucleotides (NDP, ribonucleoside diphosphate) to deoxyribonucleotides (dNDP, deoxyribonucleoside diphosphate), the DNA building blocks (dATP, dGTP, dCTP, and indirectly also dTTP) are provided. The reaction proceeds according to the following equation: NDP + NADPH + H+ → dNDP + NADP+ + H2O. The most important enzyme in this process is ribonucleotide reductase. Other involved enzymes and cofactors are thioredoxin, thioredoxin reductase with the cofactor FAD, as well as NADPH + H+.

Thioredoxin reductases catalyze the restoration of the original state of ribonucleotide reductase. What substance do they contain to fulfill this function?
- Thioredoxin reductases contain selenium in the form of selenocysteine. In the conversion of ribonucleotides to deoxyribonucleotides, they reduce thioredoxin, which in turn regenerates ribonucleotide reductase.

### Purine Metabolism

What is the basic process of de novo synthesis of purine nucleotides?
- The basic principle of the synthesis of purine nucleotides consists in the de novo synthesis of the purine ring from 5-phosphoribosyl-1α-pyrophosphate (PRPP). In ten reaction steps, PRPP is first converted into the purine nucleotide inosine-5'-monophosphate (IMP). This is then further processed in two additional steps each to the purine nucleotides adenosine-5'-monophosphate (AMP) and guanosine-5'-monophosphate (GMP), respectively.

Where does the PRPP needed for the de novo synthesis of purine nucleotides come from?
- The starting material for purine synthesis, PRPP, is synthesized from ribose-5-phosphate, which originates from the pentose phosphate pathway.

How is the de novo synthesis of purine nucleotides regulated?
- The de novo synthesis of purine nucleotides is regulated by their products and substrates: PRPP allosterically activates the key enzyme glutamine-PRPP amidotransferase, whereas adenine and guanine nucleotides inhibit it.

Which substances serve as nitrogen or carbon donors in the de novo synthesis of purine nucleotides, and where does the N9 atom of the N-glycosidic bond between the base and sugar come from?
- In the de novo synthesis of a purine nucleotide, the purine backbone is built up by nitrogen and carbon atom donors: bicarbonate, tetrahydrofolate, and the amino acids glycine, aspartate, and glutamine. The N9 atom of the N-glycosidic bond between the base and sugar of the purine nucleotide originates from glutamine.

How does the formation of IMP generally proceed and which enzyme is rate-limiting?
- Since the purine ring system cannot be enzymatically cleaved, purines are converted into uric acid and excreted in the urine in their deprotonated form (urate). For this, purine mononucleotides are first dephosphorylated to purine nucleosides, which are then split into purine bases and ribose-1-phosphate. Adenosine deaminase catalyzes the deamination of adenosine to inosine, which is converted into the purine base hypoxanthine. Hypoxanthine is degraded by xanthine oxidase via xanthine to uric acid. Guanine is degraded by guanine deaminase to xanthine and likewise to uric acid.

How does the subsequent conversion from IMP to AMP take place?
- For AMP biosynthesis, the oxygen atom at the C6 position of IMP is exchanged with an amino group. The first reaction step is catalyzed by adenylosuccinate synthetase and involves the transfer of aspartate to IMP: IMP + aspartate + GTP → adenylosuccinate + GDP + Pi. In the second step, fumarate is cleaved off, leaving only the amino group of aspartate, and adenylosuccinate is converted to AMP: adenylosuccinate → adenosine-5'-monophosphate (AMP) + fumarate.

Which reaction completes the de novo synthesis of purine nucleotides?
- In the final phase of purine nucleotide de novo synthesis, the monophosphates AMP and GMP are phosphorylated by various kinases to diphosphates (ADP, GDP) and triphosphates (ATP, GTP).

How are purine nucleotides broken down and which enzymes are necessary for this?
- Since the purine ring system cannot be enzymatically cleaved, purines are converted into uric acid and excreted in the urine as their deprotonated form (urate). For this, purine mononucleotides are first dephosphorylated to purine nucleosides, which are then split into purine bases and ribose-1-phosphate. Adenosine deaminase catalyzes the deamination of adenosine to inosine, which is converted into the purine base hypoxanthine. Hypoxanthine is degraded by xanthine oxidase via xanthine to uric acid. Guanine is degraded by guanine deaminase to xanthine and then also to uric acid.

Why can the renal manifestations of gout (urate nephrolithiasis and urate nephropathy) be countered by alkalinizing the urine?
- Because the formation of uric acid calculi in gout is promoted by a low pH value, stone formation in the urine can be counteracted by alkalinization.

What is the so-called salvage pathway, how does it roughly proceed, and what is its advantage? What clinical picture emerges when this pathway is impaired?
- The salvage pathway serves to produce purine nucleotides by reutilizing free purine bases. It is significantly less energy-consuming than the de novo synthesis of purine nucleotides. Substrates for AMP synthesis are PRPP and adenine; GMP is synthesized from PRPP and guanine; IMP from PRPP and hypoxanthine. The synthesis of GMP and IMP is catalyzed by the same enzyme: hypoxanthine-guanine phosphoribosyltransferase (HGPRT). In Lesch-Nyhan syndrome, this enzyme is missing, meaning purine bases guanine and hypoxanthine can no longer be converted into purine nucleotides. This causes an intracellular increase of PRPP, which in turn activates the key enzyme of purine base de novo synthesis and leads to accumulation of purine nucleotides. Their degradation to uric acid causes hyperuricemia, which may be accompanied by gout symptoms and neurological disorders.

### Pyrimidine Metabolism

What is the basic process of de novo synthesis of pyrimidine nucleotides?
- In the de novo synthesis of pyrimidine nucleotides, the base scaffold is first built and then connected with activated ribose phosphate. Initially, UMP (uridine-5'-monophosphate) is formed, from which CTP (cytidine-5'-triphosphate) can subsequently be synthesized via UTP: UMP → UDP → UTP → CTP. With some intermediate steps, dTMP can also be formed from dUMP.

How is the pyramiding framework newly constructed, who are the nitrogen, carbon, and oxygen donors?
- The de novo synthesis of the pyrimidine backbone occurs from aspartate and carbamoyl phosphate. However, carbamoyl phosphate must first be produced from glutamine and bicarbonate. The nitrogen, carbon, and oxygen donors therefore are glutamine (N donor: ammonia), bicarbonate (C and O donor), and aspartate (C and N donor: entire molecule). The key reaction in the de novo synthesis of pyrimidine nucleotides is the conversion of aspartate + carbamoyl phosphate → N-carbamoylaspartate + Pi. The generated N-carbamoylaspartate is then converted into orotate, which reacts with PRPP in two steps to form UMP.

Which enzyme complexes are involved in the reactions for the production of UMP?
- Three important enzyme complexes are involved in the synthesis of UMP: 1. the CAD enzyme, which itself contains three domains each with its own enzymatic activity (including a carbamoyl phosphate synthetase), 2. dihydroorotate dehydrogenase, and 3. UMP synthase.

In which other metabolic pathway is there also a carbamoyl phosphate synthetase, and what is its subcellular localization?
- In the urea cycle, mitochondrial carbamoyl phosphate synthetase 1 serves as the key enzyme. It should not be confused with cytosolic carbamoyl phosphate synthetase 2 of the CAD enzyme in pyrimidine synthesis.

Describe the synthesis of thymine-containing deoxyribonucleotides from UMP, including the involved enzyme and cofactor!
- For the production of the pyrimidine base thymine, dTMP must first be synthesized from dUMP by methylation. This reaction is catalyzed by thymidylate synthase (dTMP is also called thymidylate). As a cofactor, a derivative of folic acid (N5-N10-methylenetetrahydrofolate) serves, which transfers the methyl group. The overall reaction is: dUMP + N5-N10-methylenetetrahydrofolate → dTMP + dihydrofolate.

Name two cytostatics that interfere with pyrimidine metabolism and explain the mechanism of action of one of these two!
- Methotrexate and fluorouracil are cytostatics that interfere with pyrimidine metabolism. Methotrexate is a folate antagonist that competitively inhibits dihydrofolate reductase, thereby ultimately preventing the regeneration of N5-N10-methylenetetrahydrofolate. Fluorouracil is a pyrimidine antagonist that is intracellularly converted into fluorodeoxyuridine monophosphate (F-dUMP), which in turn inhibits thymidylate synthase. Furthermore, phosphorylated fluorouracil (fluoro-UTP) is incorporated into RNA instead of UTP.

What common intermediates are involved in the degradation of the pyrimidine nucleotides CMP and UMP?
- The final reaction steps in the degradation of CMP and UMP are uridine → uracil → dihydrouracil → β-ureidopropionate → β-alanine.