# Gene Expression and Transcription
## Summary

The genome contains the heritable information about the structure and function of a cell or organism. This information is stored in the sequence of bases in the DNA. A relatively small proportion of the DNA encodes for proteins and ribonucleic acids (RNAs), while the majority of the genome consists of sequences that cannot be assigned a clear function. The process of converting the information in the DNA into functional molecules – that is, RNA and proteins – is referred to as gene expression. The synthesis of proteins occurs in two stages: In the transcription process, a copy of the DNA is first made in the form of RNA. This copy is then used to synthesize a protein during translation.

The crucial enzymes during transcription are the DNA-dependent RNA polymerases. They synthesize RNA based on the genes encoded on the DNA, which have specific starting points (promoters) where transcription begins. Helper proteins - so-called transcription factors - are required for recognizing the promoter region. The RNA polymerase then moves along the DNA and reads one strand of the DNA double helix, on the basis of which it synthesizes RNA, until it reaches the end of the DNA segment to be read. This is marked by a termination site. In eukaryotes, the resulting primary transcript is further modified to then be available for protein synthesis, for example.

Gene expression is strictly regulated at all levels. Some genes are expressed in all cells and are known as housekeeping genes, necessary for the basic function of a cell – this is referred to as constitutive expression. Other genes, on the other hand, are only active in specific cells – their expression is regulated through various mechanisms. Genes can be activated or silenced accordingly, meaning their transcription occurs depending on the presence of specific DNA-binding proteins. However, even after transcription, the resulting mRNA can be degraded by various mechanisms before it is used for protein synthesis. Additionally, there are regulatory mechanisms at the translation level. Thus, although every cell of an organism contains the same DNA, the regulated expression of certain genes causes cells to specialize and take on very different functions, such as becoming muscle cells or liver cells.

## Overview

The DNA contains the information for the amino acid sequence of all proteins in a cell (structural genes). Furthermore, there are control elements on the DNA that allow the regulation of the expression of these structural genes.

- Gene expression: Implementation of the genetic information stored in DNA into a gene product (RNA and proteins)
- Protein biosynthesis: Central process of gene expression, consists of transcription and translation as well as their respective modifications.
    - Transcription: Synthesis of RNA based on a section of DNA in the nucleus (see: process of transcription)
        - Post-transcriptional modification
            - Processing of immature RNA (hnRNA = heterogeneous nuclear RNA) into mature RNA (mRNA = Messenger RNA) (see: Post-transcriptional modification)
            - All eukaryotic transcription products are processed, including tRNAs and rRNAs as well.
        - Transport of the mRNA into the cytoplasm
    - Translation: Translation of the base sequence of RNA into the amino acid sequence of a polypeptide in the cytosol
        - Post-translational modification: Conversion of the precursor protein into the mature protein by cleaving and attaching various residues.
- Central Dogma of Molecular Biology: States that genetic information always flows from DNA to RNA to protein and not in the other direction.
    - DNA → (Transcription) → RNA → (Translation) → Protein
    - Exception: Retroviruses that can also produce DNA from RNA using their enzyme reverse transcriptase (reverse transcription)

> [!note]
> During protein biosynthesis, the DNA is first transcribed into mRNA (transcription) and then translated into a chain of amino acids (translation)!

## Basics

The DNA serves as a template for the production of a complementary RNA molecule during transcription. Of the two strands of double-stranded DNA, only one is read.

- DNA segments
    - Matrix strand (coding strand): The DNA strand on which a complementary RNA strand is created
    - Non-template strand (coding strand): The DNA strand that is complementary to the template strand.
    - Promoter
        - Specific DNA sequence located upstream of the gene that regulates transcription
        - Binding site for RNA polymerase at the start of transcription
    - Exon-Intron Structure: Eukaryotic genes consist of alternating coding and non-coding sequences with a preceding promoter.
        - Introns: Non-coding sequences of DNA in eukaryotes
        - Exons: Coding sequences of DNA from eukaryotes
- Substrate: The nucleoside triphosphates ATP, GTP, CTP, UTP
- Enzyme: RNA polymerases
- General transcription factors: Specific auxiliary proteins that help RNA polymerases find their promoter.

### RNA Polymerases

The reactions of transcription are catalyzed by DNA-dependent RNA polymerases. In eukaryotes, there are four different types of RNA polymerases - three in the nucleus and one in the mitochondrion. They each recognize different types of promoters and thus transcribe different types of genes.

- Structure: Consists of two large subunits made up of numerous polypeptide chains.
- Functional mechanism: Synthesizing from 5' to 3', reading the DNA strand from 3' to 5'

|RNA Polymerase Type|Transcript|Location|   |
|---|---|---|---|
|RNA-Polymerase I|- rRNA (5.8S-, 18S-, and 28S-rRNA)|Nucleus|- Nucleolus|
|RNA Polymerase II|- hnRNA (predecessor of mRNA)<br>- miRNA (microRNA)<br>- snRNA (small nuclear RNA)<br>- snoRNA (small nucleolar RNA)|- Euchromatic region of the nucleus|
|RNA-Polymerase III|- tRNA<br>- rRNA (5S-rRNA)<br>- snRNA<br>- snoRNA|- Nucleolus<br>- Euchromatic region of the nucleus|
|Mitochondrial RNA Polymerase|- Mitochondrial RNA|Mitochondrion|   |

> [!NOTE]
> RNA polymerase II transcribes all genes that encode for proteins!

> [!NOTE]
> The ribosomal RNAs (rRNAs) are transcribed in the nucleolus!

### Transcription Factors

The RNA polymerases require helper proteins to locate the promoters of the genes to be transcribed. There is a distinction between general transcription factors and specific transcription factors. The transcription factors play an important role in the regulation of transcription.

- General (generic) transcription factors
    - They enable the binding of RNA polymerases to the basal promoter region by binding to specific base sequences in the chromosomal DNA.
    - Only enable the beginning of the transcription.
- Specific transcription factors
    - Modulation of transcription through binding to distal regulatory elements (so-called enhancers or silencers)
    - Additional transcription factors that specifically enhance the expression of targeted genes
    - Example: Steroid hormone receptors

#### DNA-binding Proteins

Proteins such as transcription factors that bind to DNA require special protein domains - also called structural motifs. These structural motifs typically use an α-helix or a β-sheet to bind in the major groove of the DNA. In particular, transcription factors have DNA-binding domains through which they can interact with specific DNA segments to carry out their function. By now, numerous structural motifs of such DNA-binding domains are known. Some important examples include the zinc finger domain, the leucine zipper, the helix-loop-helix motif, and the homeobox.

- Zinc finger
    - Identifier: Zinc ion, which is typically bound (coordinated) by two histidine and two cysteine residues.
    - DNA binding: Several zinc finger domains are often linked in a chain and bind with an α-helix in the major groove of the DNA.
- Leucine Zipper
    - License plate
        - Two long independent α-helices that bind together through their hydrophobic sections and form a superspiraled structure.
        - Since about every seventh amino acid residue is leucine and these residues interlock like a zipper, this structural motif is called a leucine zipper.
    - DNA Binding: The non-hydrophobic sections of the α-helices contain many basic residues (arginine, lysine) for interaction with the major groove of the DNA.
- Helix-Loop-Helix
    - License plate
        - Two polypeptide chains, which usually consist of a short and a long α-helix, are connected by a flexible loop (a loop; contains no secondary structure).
        - The two polypeptide chains dimerize over the basic sections of the α-helices.
    - DNA binding: The short basic α-helices interact with the DNA.
- Homeobox domain (with helix-turn-helix motif)
    - Characteristic: A polypeptide chain with three short α-helices arranged consecutively, where the third α-helix is oriented perpendicularly to the first two α-helices through a turn.
    - DNA binding: The third, relatively basic α-helix binds primarily as a recognition helix to bases in the major groove of the DNA.

> [!NOTE]
> An important structural motif of DNA-binding proteins is an α-helix with many basic amino acid residues!

## Process of Transcription

Transcription is divided into three phases: Initiation, Elongation, and Termination.

1. Initiation: Beginning of transcription by formation of the initiation complex and unwinding of the DNA
    1. Formation of the pre-initiation complex (also: closed initiation complex) through the binding of various general transcription factors and RNA polymerase to specific regions within the promoter (e.g. TATA box, CAAT box, GC box)

        - Example: TATA box (A/T-rich sequence within the promoter)

        1. Assembly of TFIID with the TATA box-binding protein (TBP) and other factors at the TATA region of the template strand.
        2. Stabilization through the further binding of TFIIA and TFIIB
        3. Binding of RNA polymerase II together with TFIIF to the promoter
        4. Binding of TFIIE and TFIIH
    2. Formation of the transcription bubble by unwinding the DNA double helix over a length of 10-12 bases (open initiation complex)
        - Helicasen: Components of transcription factors that catalyze DNA unwinding
        - Topoisomerases: Alter the spatial structure (the so-called topology) of DNA to relieve potential torsional stress.
    3. Start of the transcription
        - RNA synthesis begins during initiation.
2. Elongation: Elongation of the RNA strand
    - Synthesis mechanism
        1. Nucleophilic attack by the oxygen atom of the 3'OH group of the growing RNA strand on the α-phosphate of the next complementary nucleoside triphosphate (NTP) that is to be added.
        2. Formation of a phosphodiester bond
        3. Release of pyrophosphate
3. Termination
    - A number of protein factors are also required for termination.
    - Little is known about the process in eukaryotes.

> [!NOTE]
> During transcription, base pairing occurs between DNA and RNA, where in RNA, uracil (instead of thymine) pairs with adenine in DNA! Cytosine continues to pair with guanine!

> [!NOTE]
> RNA and DNA pair antiparallel: Where the 5' end is on one molecule, the 3' end is on the other molecule, and vice versa. Nevertheless, the base sequence is written in both cases in the 5'→3' direction!

> [!NOTE] Topoisomerase inhibitors
> Topoisomerases resolve the tensions that arise when the DNA double helix is wound. If they are inhibited, these tensions lead to breaks in the DNA; this prevents DNA replication and transcription. This mechanism is utilized in the treatment of malignant tumors - topoisomerase inhibitors are commonly used cytostatics! For example, the topoisomerase I inhibitor topotecan is used to treat metastatic ovarian carcinoma.

## Post-transcriptional Modification (RNA Processing)

The products of transcription are various types of RNA (see table RNA polymerases and their transcripts). One of them is hnRNA, which is synthesized by RNA polymerase II. From hnRNA, the mature mRNA is formed through post-transcriptional modifications in the nucleus. These modifications include capping, splicing, polyadenylation, and RNA editing.

### Capping

- Definition: Attaching a cap group ("Cap") to the 5' end of the hnRNA.
    - The newly formed hnRNA initially has a triphosphate at its 5' end.
- Process
    1. Cleavage of the 5'-phosphate group by an RNA triphosphatase
        - A 5'-diphosphate end is formed.
    2. Attachment of a GMP residue (formed from GTP through the cleavage of pyrophosphate) to the 5'-diphosphate end of hnRNA by a guanylyltransferase
    3. Methylation of the terminal guanine residue at position 7 and possibly of one or two ribose residues of the hnRNA using S-adenosylmethionine (SAM) as the methyl group donor.
- Result: All cap groups contain 7-methylguanylate, i.e. a guanosine with a methyl group at the N7 atom, which is bound to the 5'-ribose via three phosphate residues.
- Function
    - Protection against degradation (by exonucleases)
    - Initiation of the translation

> [!NOTE]
> The 5' end of each growing hnRNA chain and also of each mature mRNA contains a cap structure made of methylated guanine as a "cap"!

### Polyadenylation

- Definition: Attaching a tail of adenosine monophosphates (so-called polyadenylate tail) to the 3' end of hnRNA
- Process
    1. An endonuclease recognizes the polyadenylation site based on specific RNA sequences and cleaves the hnRNA there.
    2. Poly(A) polymerase binds to the junction and adds approximately 50-250 adenosine monophosphates in an ATP-dependent manner.
- Function
    - Increase in stability (protection against degradation)
    - Initiation of the translation

> [!NOTE]
> The 3' end of every mature mRNA contains a polyadenylation tail!

### Splicing ("Spleißen")

- Definition: Removal of introns from hnRNA and direct linking of exons through two transesterification reactions.
- Process
    1. Formation of the spliceosome at the exon-intron boundary
        - Spliceosome
            - Complex of:
                - Various snRNAs (small nuclear RNAs) that are bound to proteins and form the so-called snRNPs (small nuclear ribonucleoproteins)
                - Many other small proteins
                - The hnRNA to be modified
        - Involved sequence segments on the hnRNA:
            - Exon-Intron-Boundaries: Marked by specific base sequences (so-called consensus sequences) on the RNA
                - 5'-splice site
                - 3'-splice site
            - Branch point: Adenine nucleotide located within the intron, at which a lasso-like structure forms (see below)
            - Pyrimidine-rich sequence before the 3' splice site
    2. First Rescheduling
        - Nucleophilic attack of the 2'OH group on the adenine nucleotide at the branching site on the phosphodiester bond at the 5' splice site
        - Opening of the exon-intron boundary at the 5' splice site: A lasso-like structure is temporarily formed with a 2'→5' phosphodiester bond, bringing the two ends that need to be joined into close proximity to each other.
    3. Second Semester
        - Opening of the exon-intron boundary at the 3' splice site
        - Merging of the exon ends
- Function: Removing the introns, so that the resulting mRNA contains only relevant information in the form of exons.

> [!NOTE]
> The exons of a gene are the coding sections, while the introns are cut out from the hnRNA during splicing!

### RNA-Editing

- Definition: Change in the base sequence of RNA through insertion, deletion, or modification of individual bases (independent of splicing)
- Function: Additional option for the production of different proteins
- Examples
    - A-zu-I-Editing: Adenosine is deaminated to Inosine, meaning that the base Adenine is converted to Hypoxanthine.
        - Takes place on hnRNA
        - Enzyme: ADARs (Adenosine Deaminases on RNA)
            - Example: Through A-to-I editing of different subunit types of the glutamate receptor, its properties can be altered. This affects the action of glutamate in the CNS.
    - C-to-U Editing: Cytidine is deaminated to uridine, meaning that the base cytosine is converted to uracil.
        - Takes place on mRNA
        - Typical example of C-to-U editing
            - The mRNA coding for apolipoprotein B without editing is for Apo-B100.
            - With editing, it encodes for the much smaller protein Apo-B48, as the deamination of cytidine to uridine by cytidine deaminase creates a stop codon.
            - For example, enterocytes produce Apo-B48 instead of Apo-B100 compared to hepatocytes.

### Alternative Splicing

- Definition: Splicing of introns from hnRNA with variable joining of exons.
- Process: How splicing occurs, with additional splicing factors that determine the selection of splice sites.
- Function
    - From one gene, various proteins can be produced: Increasing the information density of DNA
    - The emergence of new proteins is facilitated: Faster adaptation to changing living conditions

> [!NOTE]
> The "one gene-one enzyme hypothesis" does not apply to eukaryotes! A variety of proteins can arise from a single gene through alternative splicing!

## Regulation and Inhibitors of Transcription

Both transcription and protein biosynthesis are associated with high energy consumption. Therefore, gene expression is strongly regulated. Another function of the specific regulation of gene expression is the ability to regulate the activity of gene products (mostly proteins). In this way, cells can respond to changes in their environment, such as altered nutrient supply.

For epigenetics, chromatin remodeling, and histone modification, see also: Human Genetics (Preclinical).

### Regulation of Transcription

Some genes are continuously read - this is referred to as constitutive expression. Other genes are expressed in a regulated manner. The regulation occurs through the binding of certain proteins to specific DNA sequences. Most of the time, this suppresses gene expression (repression). In some cases, however, transcription can also be enhanced by the binding of proteins to the DNA.

- Repressor: A DNA-binding protein that suppresses the transcription of a gene or operon by binding to the silencer sequences of a gene or the operator region of an operon until a signal releases this binding (negative transcriptional control).
- Activator: In the context of gene regulation, a DNA-binding protein that stimulates the transcription of a gene or operon by binding to the enhancer sequences of a gene or the promoter region of an operon, until a signal releases this binding (positive transcription control).

#### The Operon Model as an Example of Prokaryotic Gene Regulation

The regulation of expression was first studied in E. coli. Regulatory sequences in the bacterial genome ensure the expression of the gene for the enzyme β-galactosidase when the sugar lactose is available as an energy source. Additionally, other proteins are synthesized that are related to the metabolism of lactose. It is therefore a coordinated expression of several genes.

- Definition: Model for describing the mechanism of gene regulation in prokaryotes
    - An operon is a transcription unit of DNA in prokaryotes and consists of regulatory elements and several genes that code for a protein.
    - A polygenic (also polycistronic) mRNA is produced.
        - The genes in the operon are transcribed into a single mRNA.
        - All proteins of the operon are encoded for that purpose.
- Function: Adjustment to changed environmental conditions through the simultaneous increase in the expression of certain related genes.
- Example: lac operon
    - Transcription unit of genes for enzymes involved in lactose metabolism, which are expressed only in the presence of lactose.
    - Components (in the order in the genome)
        - Regulator gene lacI: Does not belong directly to the lac operon, but codes for a repressor protein that binds to the lac operator in the absence of lactose and prevents transcription.
        - Promoter: Binding site for CAP (English for catabolite activating protein; an activator protein) and RNA polymerase in transcription
        - Operator
            - Binding site for the repressor (see above)
            - The operator site overlaps with the promoter.
        - lacZ: gene for β-galactosidase
        - lacY: Gene for permease
        - lacA: Gene for transacetylase
    - Regulation
        - Presence of glucose and absence of lactose
            - The lac repressor binds to the operator, the lac operon is repressed
            - There are only very few molecules of β-galactosidase in the cell.
        - Absence of glucose, but presence of lactose
            - Lactose, or its cellular transformation product allolactose, binds to the repressor.
            - This inactivates the repressor and it no longer binds to the operator.
            - The promoter is free for the polymerase; the structural genes can be read.
            - The number of β-galactosidase molecules in the cell increases by 1000 times (Lactose serves as an inducer)

In the lac operon, the repressor binds to the operator and prevents transcription of the operon's genes in the absence of lactose.

> [!NOTE]
> Eukaryotic genes are not organized in operons!

#### Eukaryotic Gene Regulation

In eukaryotes, the regulation of gene expression is significantly more complex than in prokaryotes. This is partly because the genome of eukaryotes is much larger than that of prokaryotes. On the other hand, the DNA in the eukaryotic genome is highly condensed and packaged as chromatin within the nucleus. As a result, it is less accessible than prokaryotic DNA. However, a commonality is the great significance of activators and repressors, which bind to specific DNA sequences and either enhance or inhibit gene expression.

- Distal regulatory elements: DNA sequences that can influence the transcription rate of a gene.
    - Enhancer (engl. to enhance = "to increase")
        - Short DNA sequence of about 20bp
        - Usually a palindrome or a tandem sequence
        - Increase the transcription rate when specific transcription factors bind to them.
            - These transcription factors can exist independently or be dependent on specific ligands.
                - Ligand-dependent transcription factors: Intracellular hormone receptors that interact with enhancer sequences in the nucleus after hormone binding and increase the transcription rate of the genes they control (for an overview of hormone receptors see also: learning card signal transduction)
        - Example of an enhancer sequence: HRE (Hypoxia-Response-Element)
            - The specific transcription factor HIF-1 (hypoxia-inducible factor) binds to the HRE sequence under conditions of low oxygen (hypoxia) and induces certain target genes that are important for the hypoxic response, such as the expression of EPO and VEGF.
            - In normoxia (when oxygen is sufficiently available), the α-subunit of HIF-1 (HIF-1α) is hydroxylated by HIF prolyl hydroxylase. Hydroxy-HIF-1 is ubiquitinated and degraded in the proteasome, and cannot enhance the expression of its target genes.
    - Silencer (engl. to silence = "to dampen"): Reduce the transcription rate when specific repressors bind to them.

### Inhibitors of Transcription

Transcription inhibitors are potent cell toxins (e.g., α-Amanitin - the toxin from the death cap mushroom), but they can also be partially used as antibiotics.

|Antagonist|Mechanism of Action|Occurrence/Use|
|---|---|---|
|α-Amanitin| - Inhibits eukaryotic RNA polymerase II| - Toxin of the death cap mushroom|
|Rifampicin|- Inhibits prokaryotic RNA polymerase|- Used medically as an antibiotic|
|Actinomycin D|- Inhibits transcription by inserting itself into the DNA (intercalation) and by deforming the DNA|- Medically used as a cytostatic agent|

## Review Questions on the Chapter Gene Expression and Transcription
### Basics

How do introns differ from exons?
- Eukaryotic genes consist of alternating intron and exon sequences. Introns are the non-coding gene segments, exons the coding ones. They are transcribed continuously, i.e., the resulting RNA strand contains introns and exons. The introns are cut out in a later step, during splicing.

Which substrates does RNA polymerase incorporate during transcription?
- RNA polymerase uses the nucleoside triphosphates GTP, ATP, CTP, and UTP to synthesize RNA during transcription. It should be noted that in RNA, uracil pairs with adenine instead of thymine, while cytosine and guanine continue to pair with each other.

Where and by which enzyme is ribosomal RNA transcribed?
- All rRNA (ribosomal RNA) is transcribed in the nucleolus, but by two different enzymes: the ribosomal components 5.8S, 18S, and 28S rRNA arise from a common precursor rRNA synthesized by RNA polymerase I; the gene for 5S rRNA is transcribed by RNA polymerase III.

What is referred to as a zinc finger? What function does it have?
- The term zinc finger refers to a protein domain frequently found in transcription factors that bind in the major groove of DNA to modulate transcription. The domains bind a zinc ion in their center usually via two cysteine and two histidine residues. Often, multiple zinc finger domains are linked in series.

### Process of Transcription

What is a pre-initiation complex? Give an example!
- The pre-initiation complex forms at the beginning of the initiation phase by binding of various transcription factors and RNA polymerase II to a specific region within the promoter (e.g., to the TATA box) with a characteristic repeating AT sequence.

How do the 5' and 3' ends of DNA and RNA relate to each other?
- DNA and RNA pair antiparallel during transcription, i.e., a 5' end is connected to a 3' end. However, the base sequence is always written in 5'→3' direction.

### Post-transcriptional Modification

Which RNA is formed as a precursor for mature mRNA?
- Mature mRNA arises through post-transcriptional modifications (including capping and splicing) of the primary transcript, the hnRNA (heterogeneous nuclear RNA).

What is meant by capping? How is the "cap" structured?
- During capping, a cap group is attached to the 5' end of the hnRNA during transcription. This always contains 7-methylguanylate, i.e., a guanosine linked via three phosphate groups to the 5' ribose with an attached methyl group at the N7 atom. The capping protects hnRNA or mature mRNA from degradation by exonucleases and simultaneously serves as a signal for translation.

What process is referred to as splicing?
- To convert hnRNA into mature mRNA, introns must be removed from the hnRNA and the resulting ends joined again by two transesterification reactions. This process is called splicing.

What is a spliceosome?
- The spliceosome is the complex of snRNPs (small nuclear ribonucleoproteins), hnRNA, and other small proteins that forms at the exon-intron boundary and participates in cutting out introns and reconnecting exons.

Describe the chemical reaction mechanism of the first esterification of splicing!
- In the first of the two transesterifications of the splicing process, the 2'OH group of the adenosine nucleotide at the branch point of the intron carries out a nucleophilic attack on the phosphodiester bond at the 5' splice site.

What is meant by RNA editing? Explain it using the example of A-to-I editing!
- A-to-I editing is a type of RNA editing where the base sequence of mRNA is modified. In this case, it is a deamination process on hnRNA that converts adenosine into inosine. If this deamination occurs in a coding region, the amino acid sequence of the formed protein can be altered or a truncated protein can be produced.

What is the reason that different apolipoprotein B is produced from the same mRNA in the liver and enterocyte?
- Different proteins can arise from the same mRNA via RNA editing. In enterocytes, cytidine in apolipoprotein B mRNA is deaminated to uridine by cytidine deaminase (C-to-U editing), causing transcription to terminate here and producing apolipoprotein-B48. If, as in the liver, no mRNA modification occurs, apolipoprotein-B100 is produced.

What is referred to as alternative splicing?
- In alternative splicing, introns are removed and exons rejoined, but exons can also be treated like introns and removed. This results in different splice variants of the same gene. This increases variability and adaptability to changing environmental conditions, as well as the information density of DNA, because many different proteins can be produced from a single gene.

### Regulation and Inhibitors of Transcription

What role do distal regulatory elements play in gene expression?
- Distal regulatory elements can influence the transcription rate of a gene. They are distinguished as enhancers, which increase transcription rate, and silencers, which can inhibit it.

Explain the enhancer function of the hypoxia response elements!
- The hypoxia response element (HRE) is a specific DNA sequence in the genome and contains a binding site for the specific transcription factor HIF (hypoxia-inducible factor). Under normoxia, HIF is hydroxylated by the O2-dependent HIF prolyl hydroxylase and targeted for degradation in the proteasome. Under oxygen deficiency, HIF can bind to the HRE and induce expression of certain target genes important for the hypoxia response (e.g., VEGF, EPO).

What transcription inhibitors do you know?
- Transcription inhibitors are typically strong toxins such as α-amanitin, the poison of the death cap mushroom, which inhibits eukaryotic RNA polymerase II. Rifampicin can inhibit prokaryotic RNA polymerase and is used as an antibiotic for humans.